apps/src/code-studio/pd/application_dashboard/cohort_view_table.story.jsx

```diff
@@ -1,110 +1,108 @@
 import React from 'react';
-import {CohortViewTable} from './cohort_view_table';
-import reactBootstrapStoryDecorator from '../reactBootstrapStoryDecorator';
+import cohort_view_table, {CohortViewTable} from './cohort_view_table';
```

Comment:
i think we only want to import the named export (`CohortViewTable`) here, and we can use that for the component field you've got on line 7

Reply:
Sounds good, thanks for looking this over!